src/custom/TransferModal/TransferList/TransferList.tsx

```diff
@@ -215,7 +215,7 @@ function TransferList({
     listId?: string
   ) => (
     <StyledPaper>
-      <List dense component="div" role="list">
```

Comment:
I think it's better to use test id in meshery/meshery as the parent of this sistent element, and find them by role or aria-label. It's enough to distinguish the element i think. Wdyt?

https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-label